client/my-sites/plans-features-main/index.jsx

```diff
@@ -129,12 +142,33 @@ export class PlansFeaturesMain extends Component {
 			redirectTo,
 			siteId,
 			plansWithScroll,
-			translate,
 			customHeader,
 		} = this.props;
 
 		const plans = this.getPlansForPlanFeatures();
 		const visiblePlans = this.getVisiblePlansForPlanFeatures( plans );
+		const availablePlans = this.isDisplayingPlansNeededForFeature()
+			? visiblePlans.filter( plan => {
```

Comment:
I think we need to exclude this filter for Jetpack plans. I'm currently getting an empty page when reproducing [these steps](https://github.com/Automattic/wp-calypso/pull/41044#issuecomment-613968844).

<img width="996" alt="Screen Shot 2020-04-16 at 16 31 14" src="https://user-images.githubusercontent.com/1233880/79469170-1f32ce00-8000-11ea-9767-bde577827815.png">

Reply:
Alternatively, we could include the Jetpack plan checks (`isJetpackBusinessPlan`, `isJetpackPremiumPlan`, etc) here so we also hide Jetpack plans not containing the upsold feature/plan.

Reply:
Actually the current checks are fine. The problem is that the JP search upsell is setting a wrong link. It links to `/plans/:site?feature=search&plan=jetpack_search`. Note how the plan is `jetpack_search` while it should be `jetpack_business`.

This issue is unrelated to this PR, so I'd suggest to just skip the filter on JP plans and show all of them as previsouly.

Reply:
Thank you! The checks you suggested solve the issue with Jetpack Search whilst maintaining the checks for Jetpack nudges with a valid plan. :)  

<img width="1188" alt="Screenshot 2020-04-16 at 16 12 27" src="https://user-images.githubusercontent.com/43215253/79474477-57d1a800-7ffe-11ea-99d7-5723e81ba1db.png">